### PR TITLE
RequireJS migration: edit builds page

### PR DESCRIPTION
**Pull request by orangejenny**
@millerdev / @calellowitz

### CODE REVIEW

File: corehq/apps/builds/static/builds/js/edit-builds.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+    'knockout',
+    'underscore',
+    'hqwebapp/js/initial_page_data',
+], function (
+    $,
+    ko,
+    _,
+    initialPageData
+) {
+    var doc = initialPageData.get('doc'),
```
Comment (calellowitz) [inline corehq/apps/builds/static/builds/js/edit-builds.js]: nit: the indentation doesn't match with the next line

**Review by calellowitz — APPROVED** [review]
one minor nit, feel free to ignore but looks good.

File: corehq/apps/builds/static/builds/js/edit-builds.js (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/builds/static/builds/js/edit-builds.js]: eh? the next line is indented more because it's part of the same `var` statement

**Comment by dannyroberts** [comment]
@orangejenny hey bad news this breaks the edit builds page (confirmed with `git bisect`).

screen shot 2018-01-31 at 1 55 34 pm

**Pull request merged**